src/components/work/overview/Overview.js

```diff
@@ -51,6 +57,12 @@ export function Overview({
   // Either use type from props, or from local state
   const selectedMaterial = materialTypesMap[type] || materialTypes[0] || false;
 
+  const orderActions = {
+    onlineAccess: onOnlineAccess,
+    onLogin: onLogin,
```

Comment:
onLogin læser jeg mere som funktion der bliver kaldt, når der er logget ind - ikke én funktion som intierer login. Evt bare kald den "login"?
det samme med de to andre.
onOrder bør nok hedde openOrderModal, da den ikke udfører selve bestillingen